trajoptlib/src/error.rs

```diff
@@ -19,6 +19,8 @@ pub enum TrajoptError {
     MaxIterationsExceeded,
     #[error("The solver returned its solution so far after exceeding the maximum elapsed wall clock time.")]
     Timeout,
+    #[error("The solver returned an unparsable error code")]
```

Comment:
May as well be consistent with the other messages.
```suggestion
    #[error("The solver returned an unparsable error code.")]
```